dts/arm/nuvoton/npcx.dtsi

```diff
@@ -293,6 +293,7 @@
 		pwm0: pwm@40080000 {
 			compatible = "nuvoton,npcx-pwm";
 			reg = <0x40080000 0x2000>;
+			channel = <0>;
```

Comment:
note: channel seems to be computable like this:

```
channel = (my_pwm_addr - DT_REG_ADDR(DT_NODELABEL(pwm0))) / 0x2000;
```

(should potentially allow you to save some ROM)

Reply:
From a saving ROM usage point of view, adding a new property in DT node has no harm. There are two benefits to add a ```channel``` property.  
1. [Helper macro](https://github.com/zephyrproject-rtos/zephyr/pull/45712/files#diff-e1e5f8ebff804b124679e901b068da1bd8adf57c904dbbd6abd8c70597a89353R107) will check the peripheral node is pwm device node or not. It prevents the unexpected behavior if users enable ```open-drain``` property in the wrong pinctrl node.
2. Currently, the offset for each pwm module is 0x2000. But I cannot make sure it won't be broken if designers run out of memory space. (For example, the new pwm register address is overlapped within the other existed module.) It happened before.